### PR TITLE
form ajax

### PR DESCRIPTION
**Pull request by pnochisaki**
Implements contact form as Ajax post with associated style scaffolding and also puts the lotion in the basket!

### CODE REVIEW

File: public/js/modules/Contact.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
+var Contact = {
+	$: {
+		body: $('body'),
```
Comment (travismeyer) [inline public/js/modules/Contact.js]: remove since its not being used

File: public/js/modules/Contact.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+		contactForm: $('#contact-form')
+	},
+
+	init: function() {
+		this.contactScript();
+	},
+
+	contactScript: function() {
+		console.log('hello world');
```
Comment (travismeyer) [inline public/js/modules/Contact.js]: remove console.log

File: public/js/modules/Contact.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+		this.$.contactForm.submit(function(e) {
+	        e.preventDefault();
+	        $.ajax({
+	            url: '//formspree.io/contact@travismeyer.email',
+	            method: 'POST',
+	            data: $(this).serialize(),
+	            dataType: 'json',
+	            beforeSend: function() {
+	                $('#form-messages').append('<div class="alert alert--loading">Sending message…</div>');
```
Comment (travismeyer) [inline public/js/modules/Contact.js]: move this jquery object up to the $ object and reference it with `Contact.$.contactForm

Comment (pnochisaki) [inline public/js/modules/Contact.js]: Yeah, I tried. But all your .$.this stuff wasn't working.  And if you know anything about me, I just want to make it work.  Hehe

**Comment by travismeyer** [comment]
'ill do it myself

**Pull request merged**